x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_requirements_page/fleet_server_requirement_page.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLoadingSpinner,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiLink,
+  EuiEmptyPrompt,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+
+import { useStartServices } from '../../../hooks';
+
+export const ContentWrapper = styled(EuiFlexGroup)`
+  height: 100%;
+`;
+
+function renderOnPremInstructions() {
+  return (
+    <EuiPanel
+      grow={false}
+      hasShadow={false}
+      hasBorder={true}
+      className="eui-textCenter"
+      paddingSize="l"
+    >
+      <EuiEmptyPrompt
+        title={
+          <h2>
+            <FormattedMessage
+              id="xpack.fleet.fleetServerSetup.setupTitle"
+              defaultMessage="Add a Fleet Server"
+            />
+          </h2>
+        }
+        body={
+          <EuiText color="subdued">
```

Comment:
Nothing visually wrong with this, but this creates some unnecessary markup. The contents of  EuiEmptyPrompt's `body` prop automatically get wrapped in <EuiText> and inherit the "subdued" color. So instead, you can simply just pass in `<p>...</p>`.

![image](https://user-images.githubusercontent.com/847805/114773952-ab73f000-9d3d-11eb-8321-31769a0d6aff.png)